app/konnect/dev-portal/konnect-portal-management-automation.md

```diff
@@ -0,0 +1,132 @@
+  --header 'Authorization: kpat' \
+  --header 'Content-Type: application/json' | \
+  jq '.data | map(select(.status == "pending"))'
+```
+
+This request will return a JSON object containing developer information for developers who's `status`==`pending`:
+
+```json
+{
+    "data": [
+        {
+            "created_at": "2023-08-02T17:05:06.047Z",
+            "updated_at": "2023-08-02T17:05:06.047Z",
+            "id": "77a3da77-2b95-4b28-96ad-6112415f9620",
+            "email": "john@liverpoolfc.com",
+            "full_name": "John",
+            "status": "pending",
+            "application_count": 1
+        },
+        {
+            "created_at": "2023-08-03T23:45:50.042Z",
+            "updated_at": "2023-08-03T23:45:50.042Z",
+            "id": "b5f13fcb-190f-4418-a051-79c163770050",
+            "email": "peter@arsenal.com",
+            "full_name": "Arsenal",
+            "status": "pending",
+            "application_count": 0
+        }
+        ...
+}
+
+```
+
+Once you have obtained a list of pending developers you can begin the approval process by comparing the email address of the developer with the addresses of the developers from your trusted partner domains. Using the [Update Developer](https://docs.konghq.com/konnect/api/portal-management/latest/#/Portal%20Developers/update-developer) endpoint, you can use a PATCH request to change their status to 'approved' (or, `revoked`, `rejected`, and `pending`): 
+
+```shell
+curl --request PATCH \
+  --url https://us.api.konghq.com/v2/portals/{portal-id}/developers/{developer-id} \
+  --header 'Authorization: kpat \
+  --header 'Content-Type: application/json' \
+  --data '{"status": "approved"}'
+```
+This request can be use in combination with a list of trusted domains:
+```
+trusted_domains = ["manchestercity.com", "liverpoolfc.com", "arsenal.com"]
+```
+
+You can make sure that developers who are eligible can access your portals resources and services are given the correct access.
+
+Assigning developers to teams works in a similar way using the [Update Team Mapping](https://docs.konghq.com/konnect/api/portal-management/latest/#/Portal%20Team%20Membership/add-developer-to-portal-team) endpoint, where you can issue a POST request and add a developer to a specific team. 
+
+```shell
+curl --request POST \
+  --url https://us.api.konghq.com/v2/portals/{portal-id}/teams/{team-id}/developers \
+  --header 'Authorization: kpat \
+  --header 'Content-Type: application/json' \
+  --data '{
+  "id": "{developer-id}"
+}'
+
+```
+
+### Automation
+
+The examples above are simplified and in reality your organization may need to verify and approve developers in large quantities. You can use the API as part of your automation strategy to manage exponentially large quantities of developers who want to use your portal. Here are some recommendations on how to integrate this workflow to your developer onboarding process:
+
+1. **Continuous Integration/Continuous Deployment (CI/CD) Pipelines:**
```

Comment:
```suggestion
* **Continuous Integration/Continuous Deployment (CI/CD) Pipelines:**
```
I don't think this should be a numbered list, there's nothing that needs to be done in order, and there is no order of importance.